src/utils/decorators/custom-api-propertes.decorator.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiPropertyOptions } from '@nestjs/swagger';
```

Comment:
typo in the file name => custom-api-**propertes**.decorator.ts

Reply:
Fixed in https://github.com/gipfeli-io/gipfeli-api/pull/103